### PR TITLE
keras lecture first build

### PR DESCRIPTION
**Pull request by jstac**

### CODE REVIEW

File: lectures/keras.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+```
+
+## Models
+
+We supply functions to build two types of models.
+
+The first implements linear regression.
+
+This is achieved by constructing a neural network with just one layer, that maps
+to a single dimension (since the prediction
```
Comment (mmcky) [inline lectures/keras.md:123]: @jstac this parentheses is incomplete.

File: lectures/keras.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+
+```{code-cell} ipython3
+def build_regression_model(model):
+    model.add(Dense(units=1))
+    model.compile(optimizer=keras.optimizers.SGD(), 
+                  loss='mean_squared_error')
+    return model
+```
+
+In the function above you can see that we use stochatic gradient descent to
```
Comment (mmcky) [inline lectures/keras.md:136]:
```suggestion
In the function above you can see that we use stochastic gradient descent to
```

File: lectures/keras.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+plt.show()
+```
+
+```{code-cell} ipython3
+
+```
+
+```{code-cell} ipython3
+
+```
```
Comment (mmcky) [inline lectures/keras.md:298]:
```suggestion
```

**Review by mmcky — COMMENTED** [review]
@jstac this is a really neat lecture. 

I only have minor fixes / comments (see suggestions)

The loss figures look very similar upon casual inspection, however the y-axis scales are very different, perhaps you could have the same y-axis and plot the Epoch (x-axis) to 500 or so. Just a thought.

**Comment by mmcky** [comment]
@jstac I merged the trivial comments. Not sure what you intended to be in the parenthesis to complete. If you `git pull` and update then I'll merge and make live.

File: lectures/keras.md (hunk 1 above)
Comment (jstac) [reply lectures/keras.md:123]: Thanks @mmcky , that was meant to be "(since the prediction is real-valued)"

**Comment by jstac** [comment]
Thanks @mmcky , please make that small change above completing the sentence, make any improvements you like to the figures (or skip if you're busy) and then make it live.

**Comment by mmcky** [comment]
thanks @jstac I will make that change and make this live.
